Makefile

```diff
@@ -1,5 +1,5 @@
-VERSION ?= $(shell ./get-versions.sh VERSION)
-REVISION ?= $(shell ./get-versions.sh REVISION)
+VERSION ?= 1.2.1
```

Comment:
eh?  no?  setting version and revision from get-versions.sh is key functionality?